### DISCUSSION TITLE
Rare mobs 1-60 items

### DISCUSSION THREAD

**Discussion by Toniculus** [chromiecraft/chromiecraft#5479] [Suggestions]
there are some rare mobs that give items but a lot of them do not (cuz they have the same chane to drop an item like the regular mobs)so i suggest to give them some items that can range from 10-14 slots bag  , white/common  items with the dmg like blue weapons or armor as such (for IM players) or green/blue items like normal mobs but with a higher drop chance cuz that make players feel like killing a rare is not meaningless 

Best examples is from 60+ in outands or northrend where every  rare mob gives a specific blue item that is also powerfull depending on what level is the mob 

as another  example: Bayne is a lvl 10 rare mob in tirisfal glades that when killed has a 16% chance to give a lvl 4 cloak(ppl at lvl 10 will have the same if not even better white ccloaks ) while Deeb is a lvl 12 mob is tirisfal that as a 28% chance to give Cranial Thumper a 1h green mace that is acctally quite good for this lvl but it has a low drop chance for a rare (a similar 1h weapon can be brought from specific vendors like Feral Blade which is sold by Morhan Coppertongue in Loch Modan or 3 other NPC in the barrens)

**Comment by 55Honey** [comment]
Hello @Toniculus ,
Thanks for your suggestion.

Firstly, all creatures on ChromieCraft are supposed to give the loot, which they gave in the original 3.3.5 version of the game. Adding custom loot is not in the current scope of our project. We'd rather invest the time into making it finally right.

Rare mobs not giving the correct loot in Kalimdor and Eastern Kingdoms is a known issue. These are opened  in our issue tracker here on GitHub. Eventually a contributor will pick one of these up and propose a fix.

**Reply by Toniculus** [reply]
"Adding custom loot is not in the current scope" Sadge :(

What about increasing  the available loot for rare mobs only ? insted of a low drop chance to increase to a higher drop chance ?

Cuz lets be honnest here nobody cares if a mobs do not gives the correct loot when they drop trash items that you will sell anyways
